src/cunumeric/unary/unary_op_template.inl

```diff
@@ -64,6 +69,32 @@ struct UnaryOpImpl {
   {
     assert(false);
   }
+
+  template <CuNumericTypeCodes CODE, int DIM>
+  void operator()(UnaryOpArgs& args) const
```

Comment:
Can we move this to a separate functor instead of adding it to the existing one? I want use to avoid any accidental instantiations of this template.

Reply:
Sounds good